oob/generalities/shorthands.tex

```diff
@@ -16,14 +16,15 @@
 		\end{array} \right] \vspace{2mm} \\
 		\locPrcFlagSumCommon _{i} & \define &
 		\left[ \begin{array}{cl}
-			+ & \oobInstIsEcrecover _{i} \\
-			+ & \oobInstIsShaTwo    _{i} \\
-			+ & \oobInstIsRipemd    _{i} \\
-			+ & \oobInstIsIdentity  _{i} \\
-			+ & \oobInstIsEcadd     _{i} \\
-			+ & \oobInstIsEcmul     _{i} \\
-			+ & \oobInstIsEcpairing _{i} \\
-			+ & \locPrcFlagSumBls   _{i} \\
+			+ & \oobInstIsEcrecover _{i}            \\
+			+ & \oobInstIsShaTwo    _{i}            \\
+			+ & \oobInstIsRipemd    _{i}            \\
+			+ & \oobInstIsIdentity  _{i}            \\
+			+ & \oobInstIsEcadd     _{i}            \\
+			+ & \oobInstIsEcmul     _{i}            \\
+			+ & \oobInstIsEcpairing _{i}            \\
+			+ & \locPrcFlagSumBls   _{i}            \\
+			+ & \locFlagSumNewOsakaPrecompiles _{i} \\
```

Comment:
I would remove the `New` from the name. It won't be new for very long. E.g. call it `\locPrcFlagSumOsaka` in analogy to `\locPrcFlagSumBls`.